source/pom.xml

```diff
@@ -69,7 +69,6 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.3</version>
                 <configuration>
-                    <encoding>${source.encoding}</encoding>
```

Comment:
Do we need to change [line 12](https://github.com/simmac/threema-msgapi-sdk-java/pull/14/files#diff-bf04eeb2592d8bb10744734f5be08f88R12) from `source.encoding` to `project.build.sourceEncoding`?